src/components/modal/ModalHeader.tsx

```diff
@@ -18,9 +21,14 @@ export interface IModalHeaderDispatchProps {
 export interface IModalHeaderProps extends IModalHeaderOwnProps, IModalHeaderStateProps, IModalHeaderDispatchProps {}
 
 export class ModalHeader extends React.Component<IModalHeaderProps, {}> {
+    private canClose: boolean = true;
+
+    componentDidUpdate() {
+        this.canClose = _.isUndefined(this.props.lastOpened) ? true : this.props.lastOpened;

```

Comment:
What do you think of this syntax?: `this.canClose = _.result(this.props, 'lastOpened', true);`

Reply:
I like it! Will change.